tests/testdata/input_contracts/overflow.sol

```diff
@@ -11,7 +11,7 @@ contract Over {
   }
 
   function sendeth(address _to, uint _value) public returns (bool) {
-    require(balances[msg.sender] - _value >= 0);
+    // require(balances[msg.sender] - _value >= 0);
```

Comment:
What's this?

I suppose this require is a tautology, but you could go all out and delete it if you wanted. But isn't this line included to point out the logical error that a developer might make?

Reply:
> What's this?
 Unintended :sweat_smile: